browser-test/src/program_deep_link.test.ts

```diff
@@ -50,7 +50,7 @@ describe('navigating to a deep link', () => {
     // Assert
     await page.click('#continue-application-button')
     expect(await page.innerText('.cf-applicant-question-text')).toEqual(
-      questionText
+      questionText + "*"
```

Comment:
Note: I don't know if we handle RtL languages currently but if/when we do it'd be good to update this.